feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/swap/executor/InitialSwapFlowExecutor.kt

```diff
@@ -0,0 +1,12 @@
+package io.novafoundation.nova.feature_assets.presentation.swap.executor
+
+import io.novafoundation.nova.feature_assets.presentation.AssetsRouter
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
+import kotlinx.coroutines.CoroutineScope
+
+class InitialSwapFlowExecutor(private val assetsRouter: AssetsRouter) : SwapFlowExecutor {
+
+    override suspend fun openNextScreen(coroutineScope: CoroutineScope, chainAsset: Chain.Asset) {
+        assetsRouter.openSwapSettings(chainAsset.chainId, chainAsset.id)
```

Comment:
We can also set `chainAsset` in the `swapSettingsStore` to make it consistent with `ReselectSwapFlowExecutorFactory`

Reply:
I can't find `swapSettingsStore` entity, can you explain what you mean?

Reply:
I meant this:

![image](https://github.com/novasamatech/nova-wallet-android/assets/70131744/44ea0d09-fc80-43d3-b0a6-20945bbceae6)